MetaLamp/nft-marketplace/src/Plutus/Contracts/Services/Auction/StateMachine.hs

```diff
@@ -140,7 +141,7 @@ auctionTransition getAdditionalPayoutConstraints params@Auction{..} state@State{
             let
                 additionalConstraints = getAdditionalPayoutConstraints params state
                 constraints =
-                    Constraints.mustPayToPubKey highestBidder aAsset -- and the highest bidder the asset
+                    Constraints.mustPayToPubKey highestBidder (aAsset + minAdaTxOutValue) -- and the highest bidder the asset
```

Comment:
That seems not fair, because `minAdaTxOutValue` was first paid by the user who started auction